client_test.go

```diff
@@ -1823,72 +1823,7 @@ func Test_Client_Maintenance(t *testing.T) {
 		require.NotErrorIs(t, err, ErrNotFound) // still there
 	})
 
-	t.Run("PeriodicJobEnqueuerWithOpts", func(t *testing.T) {
-		t.Parallel()
-
-		config := newTestConfig(t, nil)
-		worker := &periodicJobWorker{}
-		AddWorker(config.Workers, worker)
-		config.PeriodicJobs = []*PeriodicJob{
-			NewPeriodicJob(cron.Every(15*time.Minute), func() (JobArgs, *InsertOpts) {
-				return periodicJobArgs{}, nil
-			}, &PeriodicJobOpts{RunOnStart: true}),
-		}
-
-		client := runNewTestClient(ctx, t, config)
-		exec := client.driver.GetExecutor()
-
-		client.testSignals.electedLeader.WaitOrTimeout()
-		svc := maintenance.GetService[*maintenance.PeriodicJobEnqueuer](client.queueMaintainer)
-		svc.TestSignals.InsertedJobs.WaitOrTimeout()
-
-		jobs, err := exec.JobGetByKindMany(ctx, []string{(periodicJobArgs{}).Kind()})
-		require.NoError(t, err)
-		require.Len(t, jobs, 1, "Expected to find exactly one job of kind: "+(periodicJobArgs{}).Kind())
-	})
-
-	t.Run("PeriodicJobEnqueuerNoOpts", func(t *testing.T) {
-		t.Parallel()
-
-		config := newTestConfig(t, nil)
-		worker := &periodicJobWorker{}
-		AddWorker(config.Workers, worker)
-		config.PeriodicJobs = []*PeriodicJob{
-			NewPeriodicJob(cron.Every(15*time.Minute), func() (JobArgs, *InsertOpts) {
-				return periodicJobArgs{}, nil
-			}, nil),
-		}
-
-		client := runNewTestClient(ctx, t, config)
-		exec := client.driver.GetExecutor()
-
-		client.testSignals.electedLeader.WaitOrTimeout()
-		svc := maintenance.GetService[*maintenance.PeriodicJobEnqueuer](client.queueMaintainer)
-		svc.TestSignals.EnteredLoop.WaitOrTimeout()
-
-		// No jobs yet because the RunOnStart option was not specified.
-		jobs, err := exec.JobGetByKindMany(ctx, []string{(periodicJobArgs{}).Kind()})
-		require.NoError(t, err)
-		require.Empty(t, jobs)
-	})
-
-	t.Run("Reindexer", func(t *testing.T) {
-		t.Parallel()
-		t.Skip("Reindexer is disabled for further development")
-
-		config := newTestConfig(t, nil)
-		config.ReindexerSchedule = cron.Every(time.Second)
-
-		client := runNewTestClient(ctx, t, config)
-
-		client.testSignals.electedLeader.WaitOrTimeout()
-		svc := maintenance.GetService[*maintenance.Reindexer](client.queueMaintainer)
-		// There are two indexes to reindex by default:
-		svc.TestSignals.Reindexed.WaitOrTimeout()
-		svc.TestSignals.Reindexed.WaitOrTimeout()
-	})
-
-	t.Run("Rescuer", func(t *testing.T) {
+	t.Run("JobRescuer", func(t *testing.T) {
```

Comment:
No substantial changes in this file despite what the diff suggests — I just hoisted the tests for the rescuer and scheduler up so that they're sorted in the same way as in the client and alphabetically.